CHANGELOG.md

```diff
@@ -7,9 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-### Fixed
-
 * Compatibility with Tarantool versions that don't support key_def and merger modules.
```

Comment:
`Compatibility with Tarantool versions that don't support key_def and merger modules.` should be in `Fixed` section. `Added support for jsonpath indexes for ...` should be in `Added`. Also I think we don't need to specify crud.select directly... because there is `crud.pairs`.